lib/cdo/log_collector.rb

```diff
@@ -38,9 +38,10 @@ def time(action_name = nil)
     )
     record_exception(e)
   end
+  alias_method :time_and_continue, :time
```

Comment:
Could you explain this to me :)?

Reply:
It is mostly for backward compatibility; I create an alias for the method instead of renaming it.

_Background:_ 
In LogCollector, `time` method measures time to execute a block and also capture any StandardError exceptions if they arise. It doesn't re-raise exception to avoid disrupting the flow of its caller.

Meanwhile, `time!` method, alias: `time_and_raise!`, will bubble up any errors occurs.